middlewares/validators/signupValidator.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable newline-per-chained-call */
+const { validateName, validateEmail, validatePassword } = require('../../lib/utils/validate');
+
+const errorMessage = (res, messages) => res.status(400).json({
+  status: false,
+  messages
+});
+
+/**
+   * @description check against errors on signup fields
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} returns error object if error exist
```

Comment:
This function returns undefined at all times

Reply:
fixed